MediaBrowser.Api/Playback/MediaInfoService.cs

```diff
@@ -234,16 +222,18 @@ public async Task<PlaybackInfoResponse> GetPlaybackInfo(GetPostedPlaybackInfo re
                         OpenToken = mediaSource.OpenToken
                     }).ConfigureAwait(false);
 
-                    info.MediaSources = new MediaSourceInfo[] { openStreamResult.MediaSource };
+                    info.MediaSources = new[] { openStreamResult.MediaSource };
                 }
             }
 
-            if (info.MediaSources != null)
+            if (info == null)
```

Comment:
```suggestion
            if (info.MediaSources == null)
```

Reply:
I also don't see why this needed to be changed. In both cases, there is `return info`, I think the previous way was more direct